### PR TITLE
[test] PrgEnv-pgi workaround for CDT 20.06

### PR DESCRIPTION
**Pull request by vkarak**
Addresses UES-928.

### CODE REVIEW

File: cscs-checks/mch/automatic_arrays_acc.py [hunk 1]
```diff
@@ -84,3 +84,8 @@ def cray_linker_workaround(self):
         if (self.current_system.name == 'dom' and
             self.current_environ.name == 'PrgEnv-cray'):
             self.variables['LINKER_X86_64'] = '/usr/bin/ld'
+
+    @rfm.run_before('compile')
+    def cdt2006_pgi_workaround(self):
+        if self.current_system.name == 'dom':
```
Comment (vkarak) [inline cscs-checks/mch/automatic_arrays_acc.py:90]: I need to further restrict it here to apply the workaround only for PrgEnv-pgi.

Comment (victorusu) [reply cscs-checks/mch/automatic_arrays_acc.py:90]: But this should be harmless to the other PE's no?

Comment (vkarak) [reply cscs-checks/mch/automatic_arrays_acc.py:90]: Yes, in fact, the problem is not due to the PR.

**Comment by vkarak** [comment]
@jenkins-cscs retry dom

**Pull request merged**